web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -141,46 +112,49 @@ const TeamReviews = ({ onBack, periodId }) => {
   const history = useHistory();
   const location = useLocation();
 
+  const [assignments, setAssignments] = useState([]);
   const [confirmOpen, setConfirmOpen] = useState(false);
   const [includeAll, setIncludeAll] = useState(false);
   const [memberFilters, setMemberFilters] = useState([]);
   const [memberSelectorOpen, setMemberSelectorOpen] = useState(false);
-  const [newRequestOpen, setNewRequestOpen] = useState(false);
   const [query, setQuery] = useState({});
+  const [reviewerSelectorOpen, setReviewerSelectorOpen] = useState(false);
   const [reviews, setReviews] = useState(null);
   const [selectedMember, setSelectedMember] = useState(null);
+  const [selectedReviewers, setSelectedReviewers] = useState([]);
   const [selfReviews, setSelfReviews] = useState({});
   const [teamMembers, setTeamMembers] = useState([]);
   const [toDelete, setToDelete] = useState(null);
 
-  const creatingReview = useRef(false);
   const loadedReviews = useRef(false);
   const loadingReviews = useRef(false);
 
   const csrf = selectCsrfToken(state);
   const currentMembers = selectCurrentMembers(state);
+  const memberMap = currentMembers.reduce((map, member) => {
+    map[member.id] = member;
+    return map;
+  }, {});
   const currentUser = selectCurrentUser(state);
   const isAdmin = selectIsAdmin(state);
-  const myTeam = selectMyTeam(state);
   const period = selectReviewPeriod(state, periodId);
-  const selectedMemberProfile = selectProfile(state, selectedMember);
-  const subordinates = selectSubordinates(state, currentUser?.id);
-
-  const handleOpenNewRequest = useCallback(
-    () => setNewRequestOpen(true),
-    [setNewRequestOpen]
-  );
-  const handleCloseNewRequest = useCallback(
-    () => setNewRequestOpen(false),
-    [setNewRequestOpen]
-  );
 
   const reviewAssignmentsUrl = '/services/review-assignments';
 
   useEffect(() => {
     loadTeamMembers();
   }, [currentMembers]);
 
+  useEffect(() => {
```

Comment:
When the circled plus icon for a member is clicked, this gets the reviewers that are currently assigned to that member so they can be passed to the `MemberSelectorDialog`.